.github/workflows/check-change-note.yml

```diff
@@ -0,0 +1,33 @@
+on:
+  pull_request_target:
+    types: [labeled, unlabeled, opened, synchronize, reopened, ready_for_review]
+    paths:
+      - "*/ql/src/**/*.ql"
+      - "*/ql/src/**/*.qll"
+      - "!**/experimental/**"
+
+jobs:
+  check-change-note:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check if change note file is present
+        uses: dorny/paths-filter@v2
+        id: paths_filter
+        with:
+          filters: |
+            change_note:
+              - '**/change-notes/*.md'
+      - name: Get PR labels
+        id: pr-labels
+        uses: joerick/pr-labels-action@v1.0.6
```

Comment:
For external actions, we always peg to a SHA, not a branch or tag. It's safer that way.

Reply:
Done!